keras_maskrcnn/bin/train.py

```diff
@@ -82,7 +82,7 @@ def create_models(backbone_retinanet, num_classes, weights, freeze_backbone=Fals
     return model, training_model, prediction_model
 
 
-def create_callbacks(model, training_model, prediction_model, validation_generator, args):
+def create_callbacks(model, training_model, prediction_model, validation_generator, args, create_evaluation=None):
```

Comment:
If we assume the arguments for this function are the same (which we do at the moment), you can also say `create_evaluation=Evaluate`; then it will default to the maskrcnn Evaluate function.

Reply:
Ohh yeah! Making the change now.